pkg/rotation/reconciler.go

```diff
@@ -251,13 +251,16 @@ func (r *Reconciler) reconcile(ctx context.Context, spcps *secretsstorev1.Secret
 	// after the provider mount request is complete
 	var requiresUpdate bool
 	var providerName string
+	podName := spcps.Status.PodName
```

Comment:
`spcps.Status.PodName` is still being referenced, let's replace all of that with `podName`. Same comment for `podNamespace` and `secretProviderClass`.